src/main/java/school/lemon/changerequest/java/introduction/hw1/Task10.java

```diff
@@ -3,7 +3,19 @@
 
 public class Task10 {
 
-    public static boolean containDigitTwo(int n) {
-        return false;
-    }
+    public static boolean containDigitTwo(int a) {
+        boolean flag = false;
+        int aLast;
+        if(a == 0)
+            flag = false;
+        while((a/10) > 0){
```

Comment:
It can be just `a > 0`. And there will be no need in additional `(a == 2)` verification.